src/opera_rtc_s1_browse/harmony_service.py

```diff
@@ -46,6 +49,12 @@ def process_item(self, item: pystac.Item, source: harmony_service_lib.message.So
                         access_token=self.message.accessToken,
                     )
 
+            if co_pol_filename is None:
+                raise ValueError(f'No co-pol tif found for {item.id}')
```

Comment:
could be a little more specific here, e.g. "No VV.tif asset found", but it's not a sticking point

Reply:
Are we eventually going to handle HH and HV?

Reply:
Maybe. I'd like to be able to handle HH+HV, as well as single-pol VV and HH, and integrate them into Worldview visualizations. I don't imagine putting any effort towards additional polarizations until a global VV+VH visualization for Oct 2023 - present is publicly available in Worldview.

Reply:
Sounds good. I updated those error messages.